sdk/lib/src/imagekitio-angular/ik-image/ik-image.component.ts

```diff
@@ -1,37 +1,45 @@
 import { Component, AfterViewInit, OnInit, ElementRef, Input, OnChanges } from '@angular/core';
 import { ImagekitService } from '../imagekit.service';
+import { Dict, QueryParameters, IkImageComponentOptions, LqipOptions } from '../utility/ik-type-def-collection'
+import { Transformation } from 'imagekit-javascript/dist/src/interfaces/Transformation';
 
 @Component({
   selector: 'ik-image',
-  template: `<img src={{src}}>`,
```

Comment:
Removing this so as to support lazy loading - otherwise this will be set by default